src/test/java/com/assignmentTest/assignment2Test.java

```diff
@@ -9,13 +9,14 @@
 
 public class assignment2Test {
```

Comment:
class name starts with capital:Assignment2Test ,and while using Junit follow naming convention of method name ie start with test (testGetReplacedStringFromOriginal).